packages/flutter_tools/doc/attach.md

```diff
@@ -7,17 +7,17 @@ without `flutter run` and provides a HotRunner (enabling hot reload/restart).
 
 ## Usage
 
-There are four ways for the attach command to discover a running app:
+There are three ways for the attach command to discover a running app:
 
 1. If the platform is Fuchsia the module name must be provided, e.g. `$
 flutter attach --module=mod_name`. This can be called either before or after
 the application is started, attach will poll the device if it cannot
 immediately discover the port
-1. On Android and iOS, just running `flutter attach` suffices. Flutter tools
+2. On Android and iOS, just running `flutter attach` suffices. Flutter tools
```

Comment:
Thanks for fixing the string `four` to `three`. However, can you leave these all as `1.`'s, so that they can be re-ordered, new ones added or existing ones removed without needing to edit each number?

I don't know if we officially follow this, but here's a markdown style guide for reference: https://google.github.io/styleguide/docguide/style.html#use-lazy-numbering-for-long-lists

Reply:
Okay I will change them to `1.` again @christopherfujino